break when getting single match information fails

sometimes I get malformed data

```
[INFO]		4686ffb9-2d1f-536b-beea-a192ed8cb402	get match info: https://www.vlr.gg/295397/tbd-gamers-club-challengers-league-2024-brazil-split-1-closed-qualifier-gf
[ERROR] ValueError: invalid literal for int() with base 10: ''
Traceback (most recent call last):
  File "/var/task/index.py", line 185, in lambda_handler
    matches = scrape_matches(page)
  File "/var/task/index.py", line 159, in scrape_matches
    match_detail = scrape_match(match_url_path)
  File "/var/task/index.py", line 120, in scrape_match
    best_of = int(best_of)
```

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -10,7 +10,8 @@
             "extensions": [
                 "amazonwebservices.aws-toolkit-vscode",
                 "ms-python.black-formatter",
-                "ms-python.isort"
+                "ms-python.isort",
+                "kddejong.vscode-cfn-lint"
             ],
             "settings": {
                 "terminal.integrated.shell.linux": "/usr/bin/zsh",
@@ -27,5 +28,6 @@
     },
     "mounts": [
         "source=${env:HOME}${env:USERPROFILE}/.aws,target=/home/vscode/.aws,type=bind,consistency=cached"
-    ]
+    ],
+    "postCreateCommand": "pip install cfn-lint"
 }

--- a/functions/sasha/index.py
+++ b/functions/sasha/index.py
@@ -153,21 +153,27 @@ def scrape_matches(page: str = 1):
     for item in html.css("a.wf-module-item"):
         match_url_path = item.attributes["href"]
 
-        sleep()
-        match_detail = scrape_match(match_url_path)
+        try:
+            sleep()
+            match_detail = scrape_match(match_url_path)
 
-        item = {
-            "id": match_detail["match_id"],
-            "eventName": match_detail["event_name"],
-            "eventCountryFlag": match_detail["event_country_flag"],
-            "startTime": match_detail["start_time"],
-            "bestOf": match_detail["best_of"],
-            "matchName": match_detail["match_name"],
-            "teams": [{"title": team} for team in match_detail["teams"]],
-            "pagePath": match_url_path,
-        }
-        logger.info("add match to the list: {}".format(item))
-        matches.append(item)
+            item = {
+                "id": match_detail["match_id"],
+                "eventName": match_detail["event_name"],
+                "eventCountryFlag": match_detail["event_country_flag"],
+                "startTime": match_detail["start_time"],
+                "bestOf": match_detail["best_of"],
+                "matchName": match_detail["match_name"],
+                "teams": [{"title": team} for team in match_detail["teams"]],
+                "pagePath": match_url_path,
+            }
+            logger.info("add match to the list: {}".format(item))
+            matches.append(item)
+        except Exception as err:
+            logger.exception(
+                "Could not get match info successfully. url: %s", match_url_path
+            )
+            continue
 
     return matches
 

